src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -1,27 +1,31 @@
 import React, { useState, useEffect, useMemo, useRef } from "react";
-import UserProfileModal from "components/organisms/UserProfileModal";
+import { useFirestoreConnect } from "react-redux-firebase";
 import { Dropdown, FormControl } from "react-bootstrap";
 import { debounce } from "lodash";
-
 import { isChatValid } from "validation";
 
 import ChatForm from "./ChatForm";
-import "./Chatbox.scss";
-import { User } from "types/User";
 import ChatMessage from "components/molecules/ChatMessage";
+import UserProfileModal from "components/organisms/UserProfileModal";
+
 import { useUser } from "hooks/useUser";
 import { useSelector } from "hooks/useSelector";
-import { useFirestoreConnect } from "react-redux-firebase";
+import { useVenueChats } from "hooks/useChats";
+
 import { WithId } from "utils/id";
-import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
 import { chatSort } from "utils/chat";
 import {
   partygoersSelector,
   partygoersSelectorData,
   privateChatsSelector,
-  venueChatsSelector,
 } from "utils/selectors";
 
+import { DEFAULT_PARTY_NAME, DEFAULT_PROFILE_IMAGE } from "settings";
+
+import { User } from "types/User";
```

Comment:
[nit] There's a specific 'import sort order' that I find useful, it generally keeps imports from 'main' 3rd party libraries at the top (eg react/redux/firestore/etc), then 'secondary' 3rd party libraries (eg. lodash/etc), then imports from our 'root' files like settings/etc near the top, then types, utils, hooks, templates, organisms, molecules, and then 'relative' imports from `../`, then `./`, and style imports are always basically the last things (or something like that.. from memory) If you look at any of the files where I have actually resorted the imports you'll probably see it.

Not a huge thing, and at some point I intend to encode this pattern via eslint or similar (as it can be a pain to think about manually sometimes), but figured I would raise it here just so you get an idea of how I like to do it.

Reply:
Would be nice to have it automatically sorted. Agreed with your approach here